base/applications/fontview/lang/cs-CZ.rc

```diff
@@ -10,6 +10,12 @@ BEGIN
     IDS_ERROR_NOMEM "K dokončení operace není dostatek paměti."
     IDS_ERROR_NOFONT "Soubor %1 není platným souborem písma."
     IDS_ERROR_NOCLASS "Inicializace okna aplikace selhala."
+    IDS_ERROR_ISINSTALLED "Tento písmo je již nainstalováno!"
+    IDS_ERROR_FONTCPY "Chyba při kopírování souboru písma!"
+    IDS_ERROR_OPENKEY "Chyba při otevírání klíče písem!"
+    IDS_ERROR_REGISTER "Chyba při registraci nového písma!"
+    IDS_SUCCESS "Úspěch"
+    IDS_COMPLETED "Instalace písma dokončena."
```

Comment:
How did you get all these different translations? Do you speak all those 20+ languages? :joy:

Reply:
fix: all translation for new added definitions turned to english